app/views/sales/new.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="sub-header">
+  <h1>Sale for <%= @product.name %> from <%= @vendor.name %></h1>
+</div>
+<%= form_for [@vendor, @product, @sale] do |f| %>
+  <%= f.label :amount, 'USD $' %>
+  <%= f.number_field :amount, :value => 0, :min => 0, :step => 0.01 %>
```

Comment:
Good use of an input type that matches very closely with the data type on our model!